sklearn/ensemble/_gb.py

```diff
@@ -1575,6 +1581,11 @@ def __init__(self, *, loss='ls', learning_rate=0.1, n_estimators=100,
             validation_fraction=validation_fraction,
             n_iter_no_change=n_iter_no_change, tol=tol, ccp_alpha=ccp_alpha)
 
+    def _validate_y(self, y):
```

Comment:
I know this does not use it, but not matching the `abstractmethod` is a little strange.

```suggestion
    def _validate_y(self, y, sample_weight=None):
```